nym-wallet/src/requests/signature.ts

```diff
@@ -0,0 +1,6 @@
+import { invokeWrapper } from './wrapper';
+
+export const sign = async (message: string): Promise<string> => invokeWrapper<string>('sign', { message });
+
+export const verify = async (publicKeyAsJson: string, signatureAsHex: string, message: string): Promise<string> =>
```

Comment:
The argument names need to match (tauri sorts out the casing differences):

```suggestion
export const verify = async (publicKeyAsJsonOrAccountAddress?: string, signatureAsHex: string, message: string): Promise<string> =>
  invokeWrapper<string>('verify', { publicKeyAsJsonOrAccountAddress, signatureAsHex, message });
```